crates/augurs-js/src/prophet.rs

```diff
@@ -442,6 +492,8 @@ impl Logs {
                         alpha0 = log.alpha0,
                         evals = log.evals,
                         notes = log.notes,
+                        "iteration {}, log_prob {:<5}, dx {:<5}, grad {:<5}, alpha {:<5}, alpha0 {:<5}, evals {:<5}, notes {}",
```

Comment:
TODO: remove this extra message line, figure out why fields in logs aren't emitted any more?

Reply:
I think it's because we're defaulting to `show_detailed_fields: false`. Might need to add a way to stop showing fields like line number and file name, probably needs a fork of `wasm-tracing`.

Reply:
Created https://github.com/dsgallups/wasm-tracing/issues/9.